src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/markduplicates/MarkDuplicatesSpark.java

```diff
@@ -40,8 +40,8 @@
             shortName = "M", fullName = "METRICS_FILE")
     protected String metricsFile;
 
-    @Argument(shortName = "DS", fullName = "duplicates_scoring_strategy", doc = "The scoring strategy for choosing the non-duplicate among candidates.")
-    public MarkDuplicatesScoringStrategy duplicatesScoringStrategy = MarkDuplicatesScoringStrategy.TOTAL_MAPPED_REFERENCE_LENGTH;
+    @Argument(shortName = "DS", fullName = "DUPLICATE_SCORING_STRATEGY", doc = "The scoring strategy for choosing the non-duplicate among candidates.")
+    public MarkDuplicatesScoringStrategy duplicatesScoringStrategy = MarkDuplicatesScoringStrategy.SUM_OF_BASE_QUALITIES;
```

Comment:
because it's the default for non-spark. It turns out that the first thing that MarkDuplicates() does is setting it and wiping out the field value (to confuse the reader, i guess). I wanted to make it easier on the eyes and set it for the field.

```
    public MarkDuplicates() {
        DUPLICATE_SCORING_STRATEGY = DuplicateScoringStrategy.ScoringStrategy.SUM_OF_BASE_QUALITIES;
    }
```